src/public/termsOfService.html

```diff
@@ -0,0 +1,36 @@
+<h1>Terms & Conditions Agreement</h1>
+<p>
+  By using the Sandclock website, dApp, and related services, I agree to the
+  <a href="/GeneralTermsOfUse.pdf">Sandclock General Terms of Use</a> and
```

Comment:
We might want to open all links in a different tab
```suggestion
  <a href="/GeneralTermsOfUse.pdf" target="_blank" rel="noopener noreferrer">Sandclock General Terms of Use</a> and
```